Podfile

```diff
@@ -52,7 +52,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-    pod 'WordPressKit', '~> 1.7.0-beta'
+    pod 'WordPressKit', '~> 1.8.0-beta.1'
```

Comment:
I'd expect this to be done with PR related to the feature that requires the change. Am I missing something?

Reply:
It looks like this change isn't in `Podfile.lock` also (that is why BuddyBuild failed)

Reply:
This was a derp. It's been removed!